#198 Load test search with JMeter

JeremiahTee opened the pull request:
This PR adds the test file used with JMeter to load test the Search feature. 
![image](https://user-images.githubusercontent.com/6520150/105568950-9f166300-5d0b-11eb-94f2-edd3bd05009f.png)


Currently, there are 6 different tests that must be run separately:
- Cell size search, 1 user
- Critical tube search, 1 user
- Specific year search and specific material, 100 users 
- 5 distinct datasets, 100 users
- 15 distinct datasets, 100 users
All of the above tests have a  10 second ramp up.

The tests display a summary report and response time graph, the findings are documented in this [Confluence article](https://databoom.atlassian.net/l/c/J5Qs6meu)

JeremiahTee committed:
add jmeter load test file for Search feature

GD-Dheer reviewed (approved):
Looks great! Nice work

Vatika1 reviewed (approved):
Good job JT :)

JeremiahTee committed:
edit tests yielding unrealistic results

upon load testing, I found that a single critical tube search takes 16 seconds. realistically, users will search with many parameters and not only by a single category. therefore it is stress tested by one user only and its result is recorded

JeremiahTee commented:
Upon load testing with 100 users, I found that some tests were unrealistic. For example, searching by cell size alone takes about 82 seconds in average - as most of the 387 queried are being returned. 
Here are screenshots of the summary reports:
Cell size search, 1 user ~ max 82 seconds
![image](https://user-images.githubusercontent.com/6520150/105569077-96725c80-5d0c-11eb-8564-1dcfd58d8b3b.png)
Critical tube search, 1 user ~ max 16 seconds 
![image](https://user-images.githubusercontent.com/6520150/105569090-aab65980-5d0c-11eb-8cc9-f00f8a40b001.png)
Specific year and specific material search, 100 users, ~ 0.6 seconds
![image](https://user-images.githubusercontent.com/6520150/105569104-dd605200-5d0c-11eb-996f-2a7ea1030248.png)
5 specific datasets, 100 users, ~ max 12 seconds
![image](https://user-images.githubusercontent.com/6520150/105569109-ecdf9b00-5d0c-11eb-8d5b-967d43f3732d.png)
15 specific datasets, 100 users ~ max 51 seconds
![image](https://user-images.githubusercontent.com/6520150/105569115-f7019980-5d0c-11eb-9e66-84613d72783c.png)

The backend was able to handle all of these requests as they pass :) 
![image](https://user-images.githubusercontent.com/6520150/105569135-2dd7af80-5d0d-11eb-91b0-026ea4d8704f.png)
